src/Tests/IndexModules/Similarity/SimilaritySettings.cs

```diff
@@ -24,21 +24,26 @@ public class Usage : PromiseUsageTestBase<ISimilarities, SimilaritiesDescriptor,
 				def = new
 				{
 					discount_overlaps = true,
-					type = "default"
+					type = "classic"
```

Comment:
👍

Reply:
This should be `default` here in 2.x, not `"classic"`

Reply:
Yep as per: https://www.elastic.co/guide/en/elasticsearch/reference/2.4/index-modules-similarity.html#default-similarity